File: Beginner/Play Piano (PLAYPIAN)/piano.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main() {

	int T;
	cin >> T;
	while(T--){
	    string s;
	    cin >> s;
	    int f=0;
	    for(int i=0; i<s.length(); i=i+2){
	        if(s[i]==s[i+1]){
	            x=1;
	            break;
	        }
	    }
	    if(f==1){
	        std::cout << "no" << std::endl;
	    }
	    else {
	        std::cout << "yes" << std::endl;
	    }
	}
	return 0;
}

File: Beginner/Play Piano (PLAYPIAN)/piano.py
t = int(input())


while t:
    P = input()
    s = 1

    for i in range(0, len(P), 2):
        if P[i]==P[i+1]:
            s = 0 
            break
        else:
            s = 1 
    if (s==0):
        print("no")
    else:
        print("yes")

    t = t-1